index.js

```diff
@@ -8,14 +8,15 @@ var gutil         = require('gulp-util')
   , temp          = require('temp')
   , spawn         = require('cross-spawn')
   , Q             = require('q')
+  , elm_make_dev  = 'node_modules/.bin/elm-make'
   , elm_make      = 'elm-make'
   , defaultArgs   = ['--yes']
   , PLUGIN        = 'gulp-elm';
 
 function processMakeOptions(options, output) {
   var args   = defaultArgs
     , ext    = '.js'
-    , exe    = elm_make;
+    , exe    = fs.existsSync(elm_make_dev) && fs.statSync(elm_make_dev).isFile() ? elm_make_dev : elm_make;

```

Comment:
I believe `fs.existsSync` is deprecated.  We should probably just pull this into a function (e.g. `getDefaultExe`), and put a `try`/`catch` around the `statSync` call (which will throw if the file doesn't exist)